extensions/ingresses/ingresses.go

```diff
@@ -105,11 +105,23 @@ func CreateIngress(client *v1.Client, ingressName string, ingressTemplate networ
 	logrus.Infof("Create Ingress: %v", ingressName)
 	ingressResp, err := client.SteveType(IngressSteveType).Create(ingressTemplate)
 	if err != nil {
-		logrus.Errorf("Failed to create ingress: %v", err)
 		return nil, err
 	}
 
-	logrus.Infof("Successfully created ingress: %v", ingressName)
+	err = kwait.PollUntilContextTimeout(context.TODO(), 500*time.Millisecond, defaults.ThirtyMinuteTimeout, true, func(ctx context.Context) (done bool, err error) {
```

Comment:
30 mins seems long for this timeout. I would think on the order of magnitude of 1 minute would suffice. WDYT?

Reply:
I can change to a minute, it was honestly copy paste from other polling.

Reply:
Resolved